_examples/http-adapter/fastly.toml

```diff
@@ -0,0 +1,13 @@
+# This file describes a Fastly Compute@Edge package. To learn more visit:
+# https://developer.fastly.com/reference/fastly-toml/
+
+authors = ["oss@fastly.com"]
+description = ""
+language = "go"
+manifest_version = 2
+name = "example"
+
+[local_server]
+  [local_server.backends]
+    [local_server.backends.ipv4]
+      url = "https://ipv4.joeshaw.org"
```

Comment:
We use `httpbin.org` for the other examples.  It has an `/ip` end-point we can use.

Reply:
👍 fixed up in 7af48a8